src/Overlay.js

```diff
@@ -48,43 +75,56 @@ class Overlay extends React.Component {
     }
 
     let child = children
+    const popperProps = {
+      placement,
+      referenceElement: this.state.target,
+      enableEvents: props.show,
+      modifiers: {
+        ...modifiers,
+        preventOverflow: {
+          padding: containerPadding || 5,
```

Comment:
this `5` should be a prop?

Reply:
I guess yeah, it's the popper default

Reply:
ah ok... we can't just use `undefined` then?